### DISCUSSION TITLE
RE2_TDB66

### DISCUSSION THREAD

**Discussion by leggomyeggroll** [praydog/REFramework#1139] [General Questions] [answered]
has anyone gotten DLSS mod to work with Re2R Dx11 non_RT version? I need help getting it to work, so the Temporal Upscaler does not appear as a option in Reframework. I have tested Mod for the Dx12 version of Re2R with RT with Reframework 1.5.8. the reason why I want trying to get the Dx11 version to work is because of the cosmetic mods I use.

**Comment by praydog** [comment, marked as answer]
It's not supported.

**Reply by leggomyeggroll** [reply]
Thanks
